composer.json

```diff
@@ -174,7 +174,7 @@
         "symfony/var-exporter": "^6.0",
         "symfony/property-info": "4.4.22 || 5.2.7",
         "symfony/serializer": "4.4.19 || 5.2.2",
-        "liip/imagine-bundle": "^2.7"
+        "liip/imagine-bundle": "^3.0.0"
```

Comment:
You can leave the conflict as 2.7.0, or bump the minimum in the require section to `^2.7.1`.  Since this bundle is a direct dependency, Composer won’t be allowed to install `>=3` right now anyway so this conflict is redundant.

Reply:
You're absolutely right. fixed

Reply:
I added 3.0.0 to enforce the incompatibility and link the issue to the commit/PR directly when it will come. I actually think it could be cool to use `^3.0.0` as a conflict for those reasons after all. WDYT

Reply:
There’s a `CONFLICTS.md` file in the repo that can be used to explain these things.  Considering 3.0 isn’t released yet, there isn’t a need to add hard conflicts to the Composer manifest yet, and anyone working on 3.0 compat would (hopefully) be aware of the B/C breaks and mitigations needed.